common/djangoapps/entitlements/tests/factories.py

```diff
@@ -1,5 +1,5 @@
 import string
-import uuid
+import uuid as uuid_tools
```

Comment:
Why make this change?

Reply:
I will remove this.  I was using uuid in the factory file I removed it and did not update the import.